src/validation.cpp

```diff
@@ -5579,21 +5576,11 @@ bool ChainstateManager::PopulateAndValidateSnapshot(
     CBlockIndex* index = nullptr;
 
     // Don't make any modifications to the genesis block.
-    // This is especially important because we don't want to erroneously
-    // apply BLOCK_ASSUMED_VALID to genesis, which would happen if we didn't skip
-    // it here (since it apparently isn't BLOCK_VALID_SCRIPTS).
```

Comment:
One useful bit of this comment might be worth preserving:

```h
// Note that the genesis block isn't BLOCK_VALID_SCRIPTS.
```

Reply:
re: https://github.com/bitcoin/bitcoin/pull/29370#discussion_r1520015210

> One useful bit of this comment might be worth preserving:

Thanks, added this back